test/basic.html

```diff
@@ -74,6 +74,22 @@
         }, 12);
       });
 
+      test('show() will accept only text and duration parameters', function() {
```

Comment:
the description is misleading (looks like you can pass only `text` and `duration` to show). What about 
> show() accepts only valid properties